### ISSUE TITLE
Error running rule validateQuoteMarks

### ISSUE THREAD

**Issue by ghlk** [jscs-dev/node-jscs#2105]
I get an error only when Error running rule validateQuoteMarks with the **-fix ** flag. When I just run it to see the list of errors, no issue, it reports the issues as expected.

Original Error messages below:
This is an issue with JSCS and not your codebase.
Please file an issue (with the stack trace below) at: https://github.com/jscs-dev/node-jscs/issues/new
TypeError: Cannot read property 'value' of undefined at base.js :
     1 |/**
--------^
     2 | * Contains all helper functions for the site that have no other home.
     3 | */

**Comment by markelog** [comment]
Can't repo it, would you mind posting your `.jscsrc` config and give a link to your file or part of it? (if you can).

**Comment by markelog** [comment]
Okay, ping us up, if you still facing the issue

**Comment by ghlk** [comment]
Oh no. So sorry Mareklog, I never saw you reply. My apologies. 
Oddly enough. My issue with validateQuoteMarks seems to have gone away.
In it's place I am getting" 
Here is my .jscsrc


```

{
  "esnext" : true,
  "disallowEmptyBlocks" : true,
  "disallowTrailingWhitespace": true,
  "disallowSpaceBeforeComma": true,
  "disallowMixedSpacesAndTabs" : true,
  "disallowSpaceBeforeComma": true, 
  "disallowSpacesInFunction": {
    "beforeOpeningRoundBrace": true,
    "beforeOpeningCurlyBrace": true
  },
  "requireTrailingComma" : {
    "ignoreSingleLine" : true
  },
  "requireCurlyBraces" : true,
  "requireCamelCaseOrUpperCaseIdentifiers": true,
  "requireParenthesesAroundIIFE": true,
  "requireCommaBeforeLineBreak": true,
  "requireDotNotation": true,
  "requireSpaceAfterComma": true,
  "requireSpacesInForStatement" : true,
  "requireSpaceAfterBinaryOperators" : true,
  "requireMultipleVarDecl": true,
  "requireSemiColons" : true,
  "requireVarDeclFirst": true,
  "validateQuoteMarks" : "'", 
  "maxErrors" : -1
}
```


I am also seeing issues with "requireSemiColons" rule. I get a line that says "Unsupported rule"

Also, I seem to not see the error with "validateQuoteMarks" when I removed "requireSemiColons" from my .jscsrc.

Hope this helps, and again sorry for the delayed response.

**Comment by markelog** [comment]
Hm, what version do you use?

**Comment by ghlk** [comment]
jscs version 2.8.0

Any other info I can give you?

**Comment by markelog** [comment]
Try to update to 2.9 version

**Comment by ghlk** [comment]
Breakthrough #1. I realized that I mispelled requireSemicolons as requireSemiColons
`"requireSemiColons" : true`
So that error is pebkac. Sorry about that.

Update to 2.9
Upon first run, I got the same "validateQuoteMarks" error. But, I haven't been able to repro that since. I have a suspicion that it has to do with the incorrect spelling of the "requireSemicolons" rule.

Sorry for all that! If I find better repro steps I'll file a new bug. Thank you so much for your time and help.
